src/sentry/monitors/logic/incident_occurrence.py

```diff
@@ -31,9 +35,6 @@ def create_incident_occurrence(
     incident: MonitorIncident,
     received: datetime | None,
 ) -> None:
-    from sentry.issues.issue_occurrence import IssueEvidence, IssueOccurrence
-    from sentry.issues.producer import PayloadType, produce_occurrence_to_kafka
```

Comment:
@wedamija any idea why this was being imported like this? I can't think of any good reason